lib/tests/streamlit/elements/arrow_dataframe_test.py

```diff
@@ -133,3 +135,31 @@ def test_pandas_version_1_3_0_and_above(self, mock_styler_translate):
 
         st._arrow_dataframe(styler)
         mock_styler_translate.assert_called_once_with(False, False)
+
+    @pytest.mark.require_snowflake()
```

Comment:
small thing: I wonder if it'd be worth it to change things so that the decorator doesn't have to be invoked as a function.

Some pytest markers like `@pytest.mark.slow` and `@pytest.mark.xfail` work like this. I think this is worth doing if we don't expect to ever have to pass arguments to this marker, but it probably doesn't matter too much either way

Reply:
Two syntaxes are supported:
- `@pytest.mark.require_snowflake()`
- `@pytest.mark.require_snowflake`
I updated the usages to use the second syntax.